### ISSUE TITLE
Size of updates

### ISSUE THREAD

**Issue by bojanv55** [shyiko/mysql-binlog-connector-java#77]
![connector](https://cloud.githubusercontent.com/assets/1422698/11625227/80fc88ba-9cd9-11e5-9fb0-901fc89e7921.png)

One question/issue regarding size of sync feed - I installed small java app (with binlog-connector) on my slave mysql server linux machine. The mysql server is syncing with the same master that java is connected to. I noticed that mysql-binlog-connector in java app is using as twice as much bandwith than the mysqld process that is doing the same sync as the slave daemon. Java app is first in the image, and mysqld is second one. You can notice that java received as twice as many data in sync process.

Do you know what could be a problem? I'm using the same user for replication.

**Comment by az3** [comment]
This project can only work with ROW type of replication. Does your server to server replication using the same format (by default it is MIXED, probably)? ROW and MIXED uses quite different output formats, so their bandwidth usage may differ due to this configuration difference.

**Comment by bojanv55** [comment]
No, as far as I can see by executing "SHOW VARIABLES LIKE 'binlog_format';" (on master and slave) it is ROW.

**Comment by shyiko** [comment]
Interesting. It might has something to do with https://dev.mysql.com/doc/internals/en/compression.html which mysql-binlog-connector-java does not turn on. Have you tried comparing numbers to `mysqlbinlog --never-stop`? Also, did you synchronized position of mysqld  & mysql-binlog-connector-java (I would appreciate if you could share the details (provisioning scripts?) on how test was conducted)?

> This project can only work with ROW type of replication.
This is actually not the case. It works fine regardless whether it's MIXED/ROW/STATEMENT.

**Comment by az3** [comment]
> It works fine regardless whether it's MIXED/ROW/STATEMENT.

Oh I missed the updates, thanks for correcting me.

**Comment by shyiko** [comment]
@bojanv55 any updates?

**Comment by bojanv55** [comment]
when I went to /var/log/mysql/* and decoded binlogs, I got full queries and not row info. So maybe in the end this mysql server is actually not doing ROW replication. But again don't know how is that possible since MASTER and SLAVE has "ROW" in settings.
